driver_test.go

```diff
@@ -1043,7 +1048,7 @@ func TestPodmanDriver_Init(t *testing.T) {
 
 	// podman maps init process to /dev/init, so we should see this
```

Comment:
You've changed the assertion but not the comment. It's not obvious to me why the assertion changed though.

Reply:
The container init binary on my system was mounted on /run/podman-init, as described on [the manpage of podman-run](https://docs.podman.io/en/latest/markdown/podman-run.1.html).